packages/codemod/changeReact.json

```diff
@@ -0,0 +1,176 @@
+[
+    {
+      "v2Name": "small=true",
+      "v3Name": "size={small}",
```

Comment:
Is it correct to use curly braces? I would've thought it would be passed in as a string, e.g.`size='small'`